float32 x
float32 y
float32 z
---
float32 joint1
float32 joint2
float32 joint3


